url-shortener/src/main/java/com/bobocode/shortener/entity/ShortenedUrl.java

```diff
@@ -0,0 +1,35 @@
+package com.bobocode.shortener.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = {"id", "shortUrl"})
+@Builder
+@ToString
+@Entity
+@Table(name = "shortened_urls")
+public class ShortenedUrl {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "short_url_gen")
+    @SequenceGenerator(name = "short_url_seq", sequenceName = "short_url_gen", schema = "romantsev", allocationSize = 1)
+    private Long id;
+
+    @Column(name = "short_url", nullable = false)
```

Comment:
@vromantsev this naming is a bit confusing. What you store in **`short_url` column is not an URL.** It's something like URL id, so it should be called accordingly.

Reply:
done